std-bits/table/src/main/java/org/enso/table/aggregations/StandardDeviation.java

```diff
@@ -48,12 +48,14 @@ public Object aggregate(List<Integer> indexes) {
         } else {
           current.count++;
           current.total += dValue;
-          current.total_sqr += dValue*dValue;
+          current.total_sqr += dValue * dValue;
         }
       }
     }
-    return current == null ? null :
-        (population ? 1 : Math.sqrt(current.count / (current.count - 1.0))) *
-            Math.sqrt(current.total_sqr / current.count - Math.pow(current.total / current.count, 2));
+
+    if (current == null) return null;
+    if (!population && current.count <= 1) return null;
```

Comment:
that looks spurious, why not just have 
```
if (current == null || (!population && ..)) return null;

Reply:
Could work too. My logic was that these two check two separate conditions - first one is for no elements at all (where I think almost all aggregations return `Nothing`) and the second one is for a single element in sample mode where the std-dev is ill-defined so we need to return `Nothing`.
I wanted to keep them separate because they are logically distinct situations.
I don't have a very strong opinion here, I think they can be merged, but also don't see why we shouldn't keep them as-is, as per my argumentation above.